code/modules/griffening_cardgame/carddatums.dm

```diff
@@ -0,0 +1,685 @@
+	Attributes = list(FACEDOWN_ONLY, SCIENTIST_REQUIRED, STOP_EQUIPPING, STOP_EFFECT, SEARCH_OPPONENT_DECK_FOR_COPY)
+
+/datum/griffeningcard/effect/enginesabotage
+
+	name = "Engine Sabotage"
+	desc = "Continuous effect, requires a antag to play, while this card is active, all opponent area cards are negated and no new area cards can be played. If there's a 'emergency shuttle' area card in play, the effects of it is not negated, but neither player can play a 'emergency shuttle' card."
+	Attributes = list(ANTAG_ONLY, CONTINUOUS, NULLIFY_OPPONENT_AREA
+
+/datum/griffeningcard/equipment/handcuffs
+
+	name = "Handcuffs"
+	desc = "Cannot be put facedown, target a human creature and equip this card to it. At the end of your third turn, the turn you play it on being the first, destroy the card. As long as that human has this card equipped, it cannot use it's effect or attack."
+	Attributes = list(NO_FACEDOWN, ANY_HUMAN, EQUIPMENT, TARGETTED, THREE_TURN_INCAPACITATE, THREE_TURN_DISCARD)
+
+/datum/griffeningcard/equipment/incendiarygrenade
+
+	name = "Incendiary Grenade"
+	desc = "If the creature this card is equipped to attacks, any opponent creature that has less than 20 DEF is destroyed. All other opponent creatures that have 20 or higher DEF will instead have DEF lowered by 20 until the end of your turn. If this effect has concluded, destroy this card."
+	REMOVEDEF = 20
+	Attributes = list(OPPONENT_REMOVEDEF_20_ONE_TURN, DISCARD_AFTER_EFFECT, EQUIPMENT, TARGETTED, DISCARD_BELOW_DEF)
+
+/datum/griffeningcard/effect/firefightinggrenade
+
+	name = "Fire Fighting Grenade"
+	desc = "This card may be played in response to Flamethrower, Plasma Fire and Incendiary Grenade. Instantly destroy both cards. When this card is activated, the opponent's turn immediately ends."
+	Attributes = list(END_OPPONENT_TURN, NULLIFY_FIRE, DISCARD_AFTER_EFFECT, ANY_TIME)
+
+/datum/griffeningcard/equipment/plasmafire
+
+	name = "Plasma Fire"
+	desc = "This card can only be played if there's a atmospheric tech in play. When this card is played, instantly reduce the DEF of all opponent humans by 10. While this card is active, all opponent humans lose 20 DEF at the start of the opponent's turn. If a creature reaches 0 DEF due to the effects of this card, the creature is killed. If any area cards are played while Plasma Fire is active, discard Plasma Fire."
+	REMOVEDEF = 10
+	Attributes = list(GLOBAL_REMOVEDEF, REMOVEDEF_20_PER_TURN, FIRE, DISCARD_ON_AREA)
+
+/datum/griffeningcard/effect/authenticationdisk
+
+	name = "Authentication Disk"
+	desc = "This card can only be played if there's a captain in play. If the captain is on your side of the field, summon two security officer immediately on your side of the field either from your hand, deck or discard pile. If the captain is on the opponent's field, immediately summon two 'nuclear operatives' from your deck, hand or discard pile."
```

Comment:
Requires Captain to play. Play two security officers immediately from your hand,deck, or discard pile. If Captain is controlled by opponent, Play two nuclear operatives from your hand, deck, or discard pile.